resources/services/backup/plans.go

```diff
@@ -0,0 +1,383 @@
+						Type:          schema.TypeJSON,
+						IgnoreInTests: true,
+					},
+					{
+						Name:        "id",
+						Description: "Uniquely identifies a rule that is used to schedule the backup of a selection of resources.",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("RuleId"),
+					},
+					{
+						Name:        "schedule_expression",
+						Description: "A cron expression in UTC specifying when Backup initiates a backup job.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "start_window_minutes",
+						Description: "A value in minutes after a backup is scheduled before a job will be canceled if it doesn't start successfully.",
+						Type:        schema.TypeBigInt,
+					},
+				},
+			},
+			{
+				Name:        "aws_backup_plan_selections",
+				Description: "Contains metadata about a BackupSelection object.",
+				Resolver:    fetchBackupSelections,
+				IgnoreError: client.IgnoreAccessDeniedServiceDisabled,
+				Columns: []schema.Column{
+					{
+						Name:        "plan_cq_id",
+						Description: "Unique CloudQuery ID of aws_backup_plan table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "creation_date",
+						Description: "The date and time a backup plan is created, in Unix format and Coordinated Universal Time (UTC).",
+						Type:        schema.TypeTimestamp,
+					},
+					{
+						Name:          "creator_request_id",
+						Description:   "A unique string that identifies the request and allows failed requests to be retried without the risk of running the operation twice.",
+						Type:          schema.TypeString,
+						IgnoreInTests: true,
+					},
+					{
+						Name:        "iam_role_arn",
+						Description: "Specifies the IAM role Amazon Resource Name (ARN) to create the target recovery point; for example, arn:aws:iam::123456789012:role/S3Access.",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("BackupSelection.IamRoleArn"),
+					},
+					{
+						Name:        "selection_id",
+						Description: "Uniquely identifies a request to assign a set of resources to a backup plan.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "selection_name",
+						Description: "The display name of a resource selection document.",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("BackupSelection.SelectionName"),
+					},
+					{
+						Name:        "conditions",
+						Description: "A list of conditions that you define to assign resources to your backup plans using tags.",
+						Type:        schema.TypeJSON,
+						Resolver:    resolveSelectionConditions,
+					},
+					{
+						Name:        "list_of_tags",
+						Description: "A list of conditions that you define to assign resources to your backup plans using tags.",
+						Type:        schema.TypeJSON,
```

Comment:
I think this should be an array rather than JSON

Reply:
I think it should be renamed to just `tags` and be kept as a json (map) to be consistent with all other resources.

Reply:
But it is explicitly just a list of strings... I would prefer that we use the most specific datatype so that we don't have to check if it is a singular value, or object. We can allow the DB to do the type checking for us

Reply:
Nevermind, you are right. It should be a json!

Reply:
I think the name as it is is correct

Reply:
Oh, I misunderstood what the issue is here.

I think the field is fine as it is. We cannot make it an array of anything. We could only make it a relation but I am not sure if it is that useful.